files/preStudies/frameworks.tex

```diff
@@ -0,0 +1,80 @@
+\section{Frameworks}
+
+This section will evaluate different frameworks for cross-platform development which could be used in this project. Some must have and nice to have capabilities will be
+	
+\textbf{Must have:}
+	\begin{itemize}
+	\item Cross platform capabilities
+	\item Wide community support
+	\item \textit{Bluetooth Low Energy} capabilities
+	\end{itemize}
+	
+\textbf{Nice to have:}
+	
+	\begin{itemize}
+	\item 1 platform to rule them all
+	\item type safety
+	\item reactive data
+	\item compiles to native code for increased performance
+	\end{itemize}
+	
+
+\subsubsection{Meteor}
+
+\textbf{language:} Javascript
+\\
+\textbf{Description:}
+\\
+\textit{Meteor} is a platform and framework for developing both web applications as well as mobile applications. It comes with \textit{Cordova} build in and is considered super fast for developing applications and prototyping. It comes with a lot of build-in tools and a package library for easy setup of things like account/user integration and login integrations of different kinds like Google and Facebook. It also comes with hot code push which is a tool for uploading new versions of your apps to the stores. \textit{Meteor} also have it's own CLI which is very useful when developing. \textit{Meteor} also allowes you to write your backend in plain javascript. Out of the box it integrates with mongo for data storage. It also uses DDP (Distributed Data Protocol) for sending data between the client and the server. This data is reactive and will be updated at each client who have subscribed to that data, if the data is changed on the server.
```

Comment:
What is Cordova? Maybe mention that it's from _Apache_.
"build-in" -> "built-in"
This sentens is very long. Maybe it could be condensed a little.

"it's" -> "its"
CLI -> Command Line Interface
javascript -> JavaScript
Explain "mongo".

The last sentence uses a lot of "data" Could be revised to: "The data is reactive and will be updated at the clients when it changes."